README.md

```diff
@@ -27,7 +27,13 @@ Demonstrate calling the method, passing in "young prince" as the argument.
 
 Write your code here:
 ```ruby
-# code here
+class BeautyBeast
+  attr_accessor :person
+def offer_rose person
+  puts "Would you take this rose,#{{person}}, in exchange for giving an old beggar woman shleter from ibtter cold?"
+end
+
+prince = BeautyBeast.new("young prince")
 ```
```

Comment:
Not sure you need a class here, just want to take your `offer_rose` method and call it so:
```ruby
def offer_rose person
   puts "Would you take this rose,#{person}, in exchange for giving an old beggar woman shleter from ibtter cold?"
end

offer_rose "young prince"
```